### ISSUE TITLE
Create a new release ?

### ISSUE THREAD

**Issue by bmm-alc** [promhippie/scw_exporter#112]
Hello,

there is a while since the last release, would it be possible to cut one please ?

Thanks

**Comment by tboerger** [comment]
Yeah but first I need to fix some api changes related to the latest client library.

**Comment by tboerger** [comment]
I would like to include https://github.com/promhippie/scw_exporter/pull/114 withion the release, after this is merged you will get a new one :)

**Comment by tboerger** [comment]
Release v1.2.0 is on the way now.
